backend/Dockerfile

```diff
@@ -1,6 +1,6 @@
 FROM ubuntu:18.04
 
-RUN apt-get update && apt-get install -y python3 python3-pip libpq-dev gettext
+RUN apt-get update && apt-get install -y python3 python3-pip libpq-dev gettext python-dev
```

Comment:
What is/was this change needed for?

Reply:
I think one of the django-cms plugins needed it but I didn't try it without it. No idea why they would need python2.